src/Materials/PBR/pbrBaseMaterial.ts

```diff
@@ -1303,7 +1303,6 @@ export abstract class PBRBaseMaterial extends PushMaterial {
             fallbacks.addFallback(fallbackRank++, "PARALLAXOCCLUSION");
         }
 
-        fallbackRank = PBRAnisotropicConfiguration.AddFallbacks(defines, fallbacks, fallbackRank);

```

Comment:
Actually it's a copy/paste problem: one of them (say the first one) should be `PBRClearCoatConfiguration.AddFallbacks(defines, fallbacks, fallbackRank);`